docs/docs-content/clusters/edge/edgeforge-workflow/prepare-user-data.md

```diff
@@ -1,40 +1,135 @@
+`user-data`, you have an additional opportunity to provide additional configuration during site deployment as
+site-specific configuration. This can replace, supplement, or override your installer configuration you provide to the
+installer ISO. For more information, refer to
 [Apply Site User Data](../site-deployment/site-installation/site-user-data.md).
 
 After installation, you can also use Local UI to make edits to the user data file before you create a cluster. Refer to
 [Edit User Data](../local-ui/host-management/edit-user-data.md) for more information.
 
-This article guides you through several important configuration blocks in the **user-data** file. However, you can use
-many additional parameters to further customize your installation. Review the Edge
-[Install Configuration](../edge-configuration/installer-reference.md) resource to learn more about all the supported
+This article guides you through several important parameters in the `.arg` file and configuration blocks in the
+`user-data` file. However, you can use many additional parameters to further customize your installation. Review the
+[Edge Artifact Build Configurations](../edgeforge-workflow/palette-canvos/arg.md) and
+[Installer Reference](../edge-configuration/installer-reference.md) resource to learn more about all the supported
 configuration parameters you can use in the configuration user data.
 
-:::tip
+The [Appliance Studio](./appliance-studio.md) is a lightweight Graphic User Interface (GUI) that allows you to design
+both files with zero risk of syntax errors and is our recommended approach to creating both files. However, since both
+`.arg` and `user-data` are plain text files, you can also manually create and edit these files once you are familiar
+with the parameters.
 
-You can also use the Operating System (OS) pack to apply additional customization using cloud-init stages. Both the Edge
-Installer configuration file and the OS pack support the usage of cloud-init stages. Refer to the
-[Cloud-Init Stages](../edge-configuration/cloud-init.md) to learn more.
+## Prerequisites
 
-:::
+<Tabs group="method">
 
-## Prerequisites
+<TabItem value="Appliance Studio">
+
+- A deployed instance of Appliance Studio. Refer to [Deploy Appliance Studio](./appliance-studio.md) to learn how to
+  deploy Appliance Studio.
+
+</TabItem>
+
+<TabItem value="Manual File Creation">
 
 - [Git](https://git-scm.com/downloads). You can ensure git installation by issuing the `git --version` command.
 
-## Prepare User Data
+</TabItem>
+
+</Tabs>
+
+## Procedure
+
+<Tabs group="method">
+
+<TabItem value="Appliance Studio">
+
+### Create .arg file
+
+1. Visit the Appliance Studio in your browser.
+
+2. Click **Design**. Then under **Argument files**, click **New .arg file**. If you have previously created and saved
+   preset, you can choose **Continue with presets**. Otherwise, click **Start from scratch**.
```

Comment:
```suggestion
2. Click **Design**. Then under **Argument files**, click **New .arg file**. If you have previously created and saved
   a preset, you can choose **Continue with presets**. Otherwise, click **Start from scratch**.
```